packages/web3-eth-ens/src/ens.ts

```diff
@@ -0,0 +1,167 @@
+import { Address } from 'web3-utils';
+import { RevertInstructionError } from 'web3-common';
+import { NonPayableCallOptions, TransactionReceipt } from 'web3-eth-contract';
+import { Registry } from './registry';
+
+export class ENS {
+	private _registryAddress: string;
+	private readonly registry: Registry;
+
+	public constructor() {
+		this.registry = new Registry();
+		this._registryAddress = ''; // TODO change this when eth.net is finished
+	}
+
+	public get registryAddress(): string {
+		return this._registryAddress;
+	}
+
+	public set registryAddress(registryAddress: string) {
+		this._registryAddress = registryAddress;
+	}
```

Comment:
If we are not processing anything extra in getters or setters, then we can simply have a public attribute `registryAddress`